card/mockCard.go

```diff
@@ -758,6 +837,78 @@ func (c *MockCard) SendPhonons(keyIndices []uint16, extendedRequest bool) (trans
 // 	return nil
 // }
 
+func (c *MockCard) PostPhonons(recipientsPublicKey []byte, nonce uint64, keyIndices []uint16) (transferPhononPackets []byte, err error) {
```

Comment:
Would probably be better to accept recipientsPublicKey as a *ecdsa.PublicKey and then convert it with FromECDSAPub within this function.